guests/site_sections/guests.py

```diff
@@ -151,7 +151,7 @@ def merch(self, session, guest_id, message='', coverage=False, warning=False, **
                 for field_name in ['country', 'region', 'zip_code', 'address1', 'address2', 'city']:
                     group_params[field_name] = params.get(field_name, '')
 
-                if not guest.info.poc_phone and not guest_merch.tax_phone:
+                if not guest.info and not guest_merch.tax_phone:
```

Comment:
This was causing a 500 error because we were trying to access a property of a model that might not exist yet. Whoops.